packages/client/bin/devnet.json

```diff
@@ -0,0 +1,800 @@
+{
+  "config": {
+    "chainId": 123456,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 60,
+    "constantinopleBlock": 80,
+    "petersburgBlock": 100,
+    "istanbulBlock": 120,
+    "berlinBlock": 140,
+    "londonBlock": 160,
```

Comment:
hm we might want to reconsider what to set these defaults since it might trip people up when hardforks suddenly change

Reply:
None, I was just using the block numbers from one of my private network files.  I'll update.